### PR TITLE
AST-634 - Elementor Pro's Header-Footer theme builder layouts does not override theme's Header-Footer builder layouts

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Elementor Pro's built header/footer layouts not override Astra's new Header-Footer builder layouts where as it works with oild header-footer layout

### Screenshots
- Issue- - https://share.bsf.io/E0uAZ5YQ
- Working with old header-footer layout - https://share.bsf.io/qGu5xj8z

### Types of changes
- Bug fix 

### How has this been tested?
- Design header-footer layouts from Elementor+Elementor pro
- Check if its override Astra's header-footer in both old & new HFB cases

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- Fix: Elementor Pro's Header-Footer theme builder layouts does not override theme's Header-Footer builder layouts.
```
Comment (premanshup) [inline changelog.txt:1]: @imnavanath This can be moved to v3.6.8 changelog below.

File: inc/core/builder/class-astra-builder-helper.php [hunk 2]
```diff
@@ -1152,3 +1152,13 @@ public static function apply_flex_based_css() {
...
+
+/**
+ * Creating global function to Check if Astra's Header-Footer builder is active.
+ *
+ * @since x.x.x
+ * @return boolean True|False
+ */
+function astra_is_header_footer_builder_active() {
+	return Astra_Builder_Helper::$is_header_footer_builder_active;
+}
```
Comment (premanshup) [inline inc/core/builder/class-astra-builder-helper.php:1164]: @imnavanath I think this global function can be removed and outer classes with method can be called something like this - 
```
\Astra_Loop::get_instance()
\Astra_Builder_Footer::get_instance()
\Astra_Builder_Helper::$is_header_footer_builder_active
```

**Review by premanshup — CHANGES_REQUESTED** [review]

File: changelog.txt (hunk 1 above)
Comment (imnavanath) [reply changelog.txt]: Done

File: inc/core/builder/class-astra-builder-helper.php (hunk 2 above)
Comment (imnavanath) [reply inc/core/builder/class-astra-builder-helper.php]: Yes it worked thanks 🎉

**Review by abhishek-bsf — APPROVED** [review]

**Review by sagarb90 — APPROVED** [review]

**Review by shubhw12 — APPROVED** [review]

**Review by premanshup — APPROVED** [review]

**Pull request merged**